truss/templates/trtllm/packages/tensorrt_llm_model_repository/postprocessing/1/model.py

```diff
@@ -32,6 +32,8 @@
 import triton_python_backend_utils as pb_utils
 from transformers import AutoTokenizer, LlamaTokenizer, T5Tokenizer
 
+INVALID_UNICODE_CHAR = "�"
```

Comment:
Good to add a reference to any huggingface docs around this, as this is a magic character.